plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/ax_sweeper.py

```diff
@@ -103,14 +114,13 @@ class AxSweeper(Sweeper):
     """Class to interface with the Ax Platform"""
 
     def __init__(
-        self, verbose_logging: bool, ax_config: DictConfig, random_seed: int,
+        self, ax_config: DictConfig,
     ):
         self.launcher: Optional[Launcher] = None
         self.job_results = None
         self.experiment = ax_config.experiment
         self.early_stopper = EarlyStopper(**ax_config.early_stop)
-        self.verbose_logging = verbose_logging
-        self.random_seed = random_seed
+        self.ax_client_config = ax_config.client
```

Comment:
might as well just store the whole config:
```
self.ax_config = ax_config
```
and use things from it directly.

Reply:
I was making the variables for different scopes (in the config). Each scope (say `experiment`, `client` etc) corresponds to one component in the system.